### PR TITLE
add banks and credit card refund API

### PR DESCRIPTION
**Pull request by LieAlbertTriAdrian**

### CODE REVIEW

File: src/examples/get_available_disbursement_banks.php [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<?php
+    require('config/xendit_php_client_config.php');
+    require('XenditPHPClient.php');
+    
+    $options['secret_api_key'] = constant('SECRET_API_KEY');
+    $options['server_domain'] = constant('SERVER_DOMAIN');
```
Comment (0505gonzalez) [inline src/examples/get_available_disbursement_banks.php]: We should not need to set the server domain. Server domain should already be part of the library. I thought we implemented this a long time ago.

File: src/examples/get_virtual_account_banks.php [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<?php
+    require('config/xendit_php_client_config.php');
+    require('XenditPHPClient.php');
+    
+    $options['secret_api_key'] = constant('SECRET_API_KEY');
+    $options['server_domain'] = constant('SERVER_DOMAIN');
```
Comment (0505gonzalez) [inline src/examples/get_virtual_account_banks.php]: Same as above

File: src/examples/issue_credit_card_refund.php [hunk 3]
```diff
@@ -0,0 +1,16 @@
+<?php
+    require('config/xendit_php_client_config.php');
+    require('XenditPHPClient.php');
+
+    $options['secret_api_key'] = constant('SECRET_API_KEY');
+    $options['server_domain'] = constant('SERVER_DOMAIN');
```
Comment (0505gonzalez) [inline src/examples/issue_credit_card_refund.php]: Same as above

File: src/examples/get_available_disbursement_banks.php (hunk 1 above)
Comment (LieAlbertTriAdrian) [inline src/examples/get_available_disbursement_banks.php]: If that's the case.

we should change this
```
        function __construct ($options) {
            if ($options['server_domain']) {
                $this->server_domain = $options['server_domain'];
            } else {
                $this->server_domain = 'https://api.xendit.co';
            }

            $this->secret_api_key = $options['secret_api_key'];
        }
```

into this
```
        function __construct ($options) {
            $this->server_domain = 'https://api.xendit.co';
            $this->secret_api_key = $options['secret_api_key'];
        }
```

What do you think?

Comment (LieAlbertTriAdrian) [inline src/examples/get_available_disbursement_banks.php]: @0505gonzalez

**Comment by 0505gonzalez** [comment]
LGTM

**Pull request merged**